variables.tf

```diff
@@ -109,6 +121,12 @@ variable "email_sending_domains" {
   }
 }
 
+variable "findings_data_bucket_name" {
```

Comment:
This is an ugly suggestion that I thought should at least be mentioned: If this being an empty string is an invalid resource should we use the `count` meta-argument hack to only create resources around this variable's use if this variable is not empty? Something like:


```terraform
count = (var.findings_data_bucket_name != "") ? 1 : 0
```

Reply:
In the case of an empty string, the policy that gets created is accepted as valid by Terraform and AWS IAM.  At the same time, the resource pointed to be the policy is invalid; therefore, there is zero possibility that the policy could accidentally allow access to an existing resource.  So I think this is OK as it is and there is no need to introduce the additional complication.  I'm open to being convinced otherwise, though.